tls/s2n_ktls.c

```diff
@@ -160,6 +160,7 @@ static S2N_RESULT s2n_ktls_crypto_info_init(struct s2n_connection *conn, s2n_ktl
 
     switch (conn->actual_protocol_version) {
         case S2N_TLS12:
+            RESULT_ENSURE(conn->prf_space, S2N_ERR_INVALID_STATE);
```

Comment:
Should we add the check for both versions so that https://github.com/aws/s2n-tls/issues/3206 isn't a potentially breaking change?

Reply:
Good point. And it probably shouldn't behave differently depending on the version anyway-- that could be confusing for users.